tests/resources/OmniFixture-dump.sql

```diff
@@ -1256,6 +1256,8 @@ CREATE TABLE `user` (
   `superuser_access` tinyint(2) unsigned NOT NULL DEFAULT '0',
   `date_registered` timestamp NULL DEFAULT NULL,
   `ts_password_modified` timestamp NULL DEFAULT NULL,
+  `invite_status` varchar(40) DEFAULT NULL,
+  `invite_at` timestamp NULL DEFAULT NULL,
```

Comment:
That should actually not be needed. When the fixture is set up it tries to perform any available updates that were added after the Omnifixture was last created (based on the version in the option table). So as long as you have added an update script and the version in `core\Version.php` has been increased, the columns should be added automatically